zhaquirks/tuya/ts0601_pool_sensor.py

```diff
@@ -0,0 +1,276 @@
+
+        if force_new_interval and self.next_refresh_interval > 0:
+            self.debug(
+                "using refresh interval of %d minutes", self.next_refresh_interval
+            )
+            self._update_timer_handle = self._loop.call_later(
+                self.next_refresh_interval, self.handle_auto_update_timer_wrapper
+            )
+
+    def handle_auto_update_check_change(self):
+        """Auto update schedule next interval check."""
+        self.handle_auto_update_setup_next_call()
+        self._loop.call_later(self.check_interval, self.handle_auto_update_check_change)
+
+    def handle_auto_update_timer_wrapper(self):
+        """Auto update handle refresh and schedule next update."""
+        self.create_catching_task(self.handle_auto_update())
+        self.handle_auto_update_setup_next_call(force_new_interval=True)
+
+    async def handle_auto_update(self):
+        """Auto update invoke data refresh command."""
+        tuya_cluster = self.endpoint.device.endpoints[1].in_clusters[
+            TuyaMCUCluster.cluster_id
+        ]
+        self.debug("sending refresh query command")
+        await tuya_cluster.command(TUYA_QUERY_DATA)
+
+
+(
+    TuyaQuirkBuilder("_TZE200_v1jqz5cy", "TS0601")
+    .tuya_enchantment(read_attr_spell=True, data_query_spell=True)
+    .tuya_battery(
+        dp_id=7, battery_type=BatterySize.Built_in, battery_qty=4, battery_voltage=36
+    )
+    .tuya_temperature(dp_id=2, scale=10)
+    .tuya_sensor(
+        dp_id=10,
+        attribute_name="ph_measured_value",
+        divisor=100,
+        type=t.uint16_t,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PH,
+        translation_key="ph_measured_value",
+        fallback_name="pH",
+    )
+    .tuya_sensor(
+        dp_id=1,
+        attribute_name="total_dissolved_solids",
+        type=t.uint16_t,
+        unit=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="total_dissolved_solids",
+        fallback_name="Total dissolved solids",
+    )
+    .tuya_sensor(
+        dp_id=11,
+        attribute_name="ec_measured_value",
+        type=t.uint16_t,
+        unit=CONDUCTIVITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="ec_measured_value",
+        fallback_name="Electrical conductivity",
+    )
+    .tuya_sensor(
+        dp_id=117,
+        attribute_name="salt_measured_value",
+        type=t.uint16_t,
+        unit=CONCENTRATION_PARTS_PER_MILLION,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="salt_measured_value",
+        fallback_name="Salt concentration",
+    )
+    .tuya_sensor(
+        dp_id=101,
+        attribute_name="redox_potential",
+        type=t.uint16_t,
+        unit=UnitOfElectricPotential.MILLIVOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="redox_potential",
+        fallback_name="ORP level",
+    )
```

Comment:
You've set a device class for the "ORP number config entities", so I think we should also add one here.
I think `SensorDeviceClass.VOLTAGE` makes sense here as well?